packages/cli/src/Studio.ts

```diff
@@ -93,6 +94,11 @@ ${bold('Examples')}
 
     const staticAssetDir = path.resolve(__dirname, '../build/public')
 
+    const schema = await fs.promises.readFile(schemaPath, 'utf-8')
+    const config = await getConfig({ datamodel: schema, ignoreEnvVarErrors: false })
+    const _directUrl = getDirectUrl(config.datasources[0])
+    const directUrl = _directUrl.value
```

Comment:
This retrieves the `directUrl` explicitly from the schema and puts it into `StudioServer` to be used.

(This is a bit dirty, as Studio gets the `url` value by itself, but this was the best and most straightforward way to make Studio actually handle `directUrl` without completely refactoring it and the way it uses Prisma Client.)

Reply:
Simpler is better.